README.md

```diff
@@ -0,0 +1,57 @@
+# Sugar 🇦🇮
+
+Sugar AI is a **central hub for all AI-powered chatbots and assistants** integrated within Sugar. Designed to enhance interactive learning, it **brings the power of AI and LLMs** into various educational activities, making learning more engaging and accessible for children.
+By seamlessly integrating AI into Sugar activities, Sugar AI fosters creativity, problem-solving, and interactive learning. Children can engage with AI chatbots for conversations, receive coding assistance through intelligent programming guides, and explore new concepts in an intuitive and friendly environment.
+
+## Current AI Models🤖
+
+### 1. Chatbot for the [Chat Activity](https://github.com/sugarlabs/chat)💬
+An interactive AI that helps kids learn through engaging conversations.
```

Comment:
Please make this a complete sentence (needs subject and verb)

e.g. The Chat Activity chatbot is an interactive gen-AI bot that helps kids learn through engaging conversations.